src/sanitizer.js

```diff
@@ -106,6 +107,10 @@ const WHITELISTED_ATTRS = [
   'href',
   'on',
   'placeholder',
+  /* Attributes added for amp-bind */
+  // TODO(kmh287): Add more whitelisted attributes for bind?
+  'text',
+  'class',
```

Comment:
Does this result in `<p text="foo" class="bar">` being valid (in addition to `[text]` and `[class]`)?